xsd/netex_part_2/part2_journeyTimes/netex_vehicleJourneyFrequency_version.xsd

```diff
@@ -212,26 +212,52 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>Elements for a JOURNEY FREQUENCY GROUP.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:sequence>
-			<xsd:element name="FirstDepartureTime" type="xsd:time">
-				<xsd:annotation>
-					<xsd:documentation>Time of first departure in JOURNEY FREQUENCY GROUP.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="FirstDayOffset" type="DayOffsetType" default="0" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Offset days for end time. Number of days after the starting operational day if journey if  not same calendar day. Default is 0 for same day.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="LastDepartureTime" type="xsd:time" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Time of last departure in JOURNEY FREQUENCY GROUP.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="LastDayOffset" type="DayOffsetType" default="0" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Offset days for end time. Number of days after the starting departure time of the journey if  not same calendar day. Default is 0 for same day.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
+			<xsd:choice>
+				<xsd:sequence>
+					<xsd:element name="FirstDepartureTime" type="xsd:time">
+						<xsd:annotation>
+							<xsd:documentation>Time of first departure in JOURNEY FREQUENCY GROUP.</xsd:documentation>
+						</xsd:annotation>
+					</xsd:element>
+					<xsd:element name="FirstDayOffset" type="DayOffsetType" default="0" minOccurs="0">
+						<xsd:annotation>
+							<xsd:documentation>Offset days for end time. Number of days after the starting operational day if journey if  not same calendar day. Default is 0 for same day.</xsd:documentation>
```

Comment:
For me the term *end time* is not clear - but I guess that it makes sense to everyone else who is a more seasoned expert?

Reply:
This is not clear. I would even argue it is actually wrong.

Reply:
was there before..... changed it now. better?